ChemistryLib/PhreeqcIO.h

```diff
@@ -80,6 +80,6 @@ class PhreeqcIO
     std::unique_ptr<Output> const _output;
     std::vector<std::pair<int, std::string>> const&
         _process_id_to_component_name_map;
-    double _dt;
+    double _dt = std::numeric_limits<double>::quiet_NaN();
```

Comment:
It it better to move this to the initializer list of the ctor. The same for other similar changes.

Reply:
To be discussed. Class member initialization also has advantages.

Reply:
Since the member is not an instance of object, it is OK to set it as in class initialization.